iquip/apps/scheduler.py

```diff
@@ -0,0 +1,298 @@
+"""Protocol module for defining common forms."""
```

Comment:
First of all, all developers should follow several coding rules!

- After the module docstring, add a single empty line.
- After finishing to import modules, add a single empty line. (You got it right!)
- Between the outmost classes or functions, add two empty lines.
- After the class docstring, add a single empty line.
- Indicate a type hinting for each parameter and return value.

Reply:
Also, you should meet the Pylint condition!